services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -231,18 +256,204 @@ export const DocumentTable = ({
   const minimalProps = isMinimalView
     ? { size: "small" as SizeType, rowClassName: "ant-table-row-minimal" }
     : null;
+
+  const handleRowSelectionChange = (value) => {
+    if (documentTypeCode === "") {
+      setDocumentTypeCode(value[0].major_mine_application_document_type_code);
+    }
+
+    const documentManagerGuids = value
+      .filter((row) => row.versions !== undefined)
```

Comment:
Do we need to also check that row.versions is not null here too ?

Reply:
Oh I think this is changed now, it should look like this with the recent changes:

```
    const documentManagerGuids = rowSelection
      .filter((row) => row.is_latest_version)
```